### PR TITLE
add led and echo support

### PR DESCRIPTION
**Pull request by akarnil**

### CODE REVIEW

File: samples/rx65ncloudkit/basic-sample/src/iotconnect_app.c [hunk 1]
```diff
@@ -42,6 +42,22 @@ void memory_test() {
 #endif /* MEMORY_TEST */
 
 
+#define STRINGS_ARE_EQUAL 0
+#define FREE(x) if ((x)) { free((void*)x); (x) = NULL; }
+#define DOES_COMMAND_MATCH(input_str, command_enum) (strncmp((input_str), command_strings[(command_enum)], strlen(command_strings[(command_enum)])) == STRINGS_ARE_EQUAL)
```
Comment (nmatthews-witekio) [inline samples/rx65ncloudkit/basic-sample/src/iotconnect_app.c]: Really not a great fan of the macros here.

File: samples/rx65ncloudkit/basic-sample/src/iotconnect_app.c [hunk 2]
```diff
@@ -42,6 +42,22 @@ void memory_test() {
...
+    COMMAND_MAX
+} command_type_t;
+
+const char *command_strings[] = {
+    [ECHO] = "echo ",
+    [LED] = "led ",
+};
+
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wunused-function"
```
Comment (nmatthews-witekio) [inline samples/rx65ncloudkit/basic-sample/src/iotconnect_app.c]: Hmmm, turning gcc warnings off?

File: samples/rx65ncloudkit/basic-sample/src/iotconnect_app.c [hunk 3]
```diff
@@ -69,33 +85,93 @@ static void on_ota(IotclEventData data) {
...
+    const char *str = iotcl_create_serialized_string(msg, false);
+    iotcl_telemetry_destroy(msg);
+    printf("Sending: %s\r\n", str);
+    iotconnect_sdk_send_packet(str); // underlying code will report an error
+    iotcl_destroy_serialized(str);
+}
+
+static int get_command_type(const char* command_str) {
+
+
```
Comment (nmatthews-witekio) [inline samples/rx65ncloudkit/basic-sample/src/iotconnect_app.c]: Could have a for loop over the command_strings[] ??

File: samples/rx65ncloudkit/basic-sample/src/iotconnect_app.c (hunk 2 above)
Comment (akarnil) [reply samples/rx65ncloudkit/basic-sample/src/iotconnect_app.c]: sorted

File: samples/rx65ncloudkit/basic-sample/src/iotconnect_app.c (hunk 1 above)
Comment (akarnil) [reply samples/rx65ncloudkit/basic-sample/src/iotconnect_app.c]: keeping only FREE because the others can be re-implemented another way

**Review by nmatthews-witekio — APPROVED** [review]

**Pull request merged**